store/loomethdb.go

```diff
@@ -192,7 +184,7 @@ func (s *LoomEthdb) NewLogBatch(logContext *ethdbLogContext) ethdb.Batch {
 	if !loggerStarted {
 		file, err := os.Create(b.params.LogFilename)
 		if err != nil {
-			return &b.batch
+			return b
```

Comment:
This error handling doesn't make much sense to me

Reply:
panic instead then